reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,144 @@
+---
+ms.date:  2017-12-04
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell
+title:  about_Logging
+---
+
+# About Logging
+
+## Short Description
+
+PowerShell logs details of PowerShell operations, such as starting
+and stopping the program engine and starting and stopping PowerShell
+providers.  it will also log details about PowerShell commands.
+
+## Long Description
+
+The location of PowerShell logs is dependent on the target platform.
```

Comment:
The `Long Description` should be a more detailed version of the `Short Description` rather than something different.  I think you should move the contents of `Short Description` down here and make `Short Description` a single sentence like: PowerShell logs internal operations including the engine, providers, and cmdlets.

Reply:
updated